wallet/wallet.go

```diff
@@ -87,7 +87,8 @@ type Wallet struct {
 
 	// Handlers for stake system.
 	stakeSettingsLock  sync.Mutex
-	voteBits           stake.VoteBits
+	defaultVoteBits    stake.VoteBits
+	ticketsVoteBits    map[string]stake.VoteBits
```

Comment:
i'd not save these in memory, and it will make it easier to deal with not having an api to read all of the ticket prefs together at startup.

Reply:
I was thinking caching these has the added benefit of not requiring db reads before voting on _each ticket_ similar to the current caching mechanism that ensures voting can take place without reading prefs from the db. I'm uncaching as requested but probably something to (re)consider?

Reply:
Per chat discussion, db reads are fast and no cause for concern.